lib/rack/tracker.rb

```diff
@@ -55,7 +55,11 @@ def html?; @headers['Content-Type'] =~ /html/; end
 
     def inject(env, response)
       @handlers.each(env) do |handler|
-        response.gsub!(%r{</#{handler.position}>}, handler.render + "</#{handler.position}>")
+        if handler.container_position == :closing
+          response.gsub!(%r{</#{handler.container_tag}>}, handler.render + "</#{handler.container_tag}>")
```

Comment:
a17b6c0bd8a328ee35071a5dae68ea1ce48d7475 should take care of the element with attributes issue